src/kernel-install/kernel-install.c

```diff
@@ -1487,6 +1504,9 @@ static int help(void) {
                "     --root=PATH               Operate on an alternate filesystem root\n"
                "     --image=PATH              Operate on disk image as filesystem root\n"
                "     --image-policy=POLICY     Specify disk image dissection policy\n"
+               "     --entry-type=type1|type2|all\n"
+               "                               Operate only on the specified bootloader\n"
+               "                               entry type\n"
```

Comment:
This is somewhat misleading, because the setting only has effect on `remove`, at least in our code. But I guess it's fine to leave this ambiguous, we might add further support later on.

Reply:
Thank you for all the suggestions.